openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/MultiPipeline.spec.ts

```diff
@@ -0,0 +1,153 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { expect, test } from '@playwright/test';
+import { TableClass } from '../../support/entity/TableClass';
+import { getApiContext, redirectToHomePage, uuid } from '../../utils/common';
+
+// use the admin user to login
+test.use({ storageState: 'playwright/.auth/admin.json' });
+
+test('MultiPipeline', async ({ page }) => {
```

Comment:
Can we make it more description a bit, Can't figure out from title about what type of test it is